multipy/runtime/interpreter/CMakeLists.txt

```diff
@@ -47,17 +49,24 @@ include(GoogleTest)
 # the modules in a strange nested path, and then that path is relative to the
 # Cmake ExternalProject root in the cmake build dir.
 ExternalProject_Get_property(cpython SOURCE_DIR)
-SET(PYTHON_MODULE_DIR "${SOURCE_DIR}/build/temp.linux-x86_64-3.8/${SOURCE_DIR}/Modules")
+SET(PYTHON_MODULE_DIR "${SOURCE_DIR}/build/temp.linux-x86_64-{CPYTHON_VERSION}/${SOURCE_DIR}/Modules")
 SET(PYTHON_STDLIB_DIR "${SOURCE_DIR}/Lib")
-SET(PYTHON_STDLIB "${PYTHON_INSTALL_DIR}/lib/libpython_stdlib3.8.a")
+SET(PYTHON_STDLIB "${PYTHON_INSTALL_DIR}/lib/libpython_stdlib{CPYTHON_VERSION}.a")
+
+if(${CPYTHON_VERSION} MATCHES "3\.[8-9]")
```

Comment:
I feel like later on versioning updates to python might be annoying, it may be a good idea to document the process or turn this logic into a function to make the process a bit easier. Otherwise, lgtm.